Tone/instrument/NoiseSynth.ts

```diff
@@ -0,0 +1,127 @@
+
+import { AmplitudeEnvelope } from "../component/envelope/AmplitudeEnvelope";
+import { NormalRange, Time } from "../core/type/Units";
+import { deepMerge, optionsFromArguments } from "../core/util/Defaults";
+import { RecursivePartial } from "../core/util/Interface";
+import { Noise, NoiseType } from "../source/Noise";
+import { Instrument } from "./Instrument";
+import { Synth, SynthOptions } from "./Synth";
+
+interface NoiseSynthOptions extends SynthOptions {
```

Comment:
Since the NoiseSynth extends `Instrument`, its options should extend InstrumentOption instead of SynthOptions.